.github/workflows/sync.yml

```diff
@@ -27,7 +27,7 @@ jobs:
           else
             curl -O https://nf-co.re/pipeline_names.json
           fi
-          echo "name=matrix::$(cat pipeline_names.json)" >> $GITHUB_OUTPUT
+          echo "matrix==$(cat pipeline_names.json)" >> $GITHUB_OUTPUT
```

Comment:
do we need a comparison here

Reply:
whoops no! It was a typo
I am basing the change on [these docs](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/), if you want to double check